modules/drivers/presto-jdbc/src/metabase/driver/presto_jdbc.clj

```diff
@@ -56,7 +56,7 @@
 
 ;;; Presto API helpers
 
-(def presto-type->base-type
+(def ^:private ^{:arglists '([column-type])} presto-type->base-type
```

Comment:
Not strictly related but a lot of this stuff should have been `^:private` so I fixed it